cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/metrics.rst

```diff
@@ -560,20 +590,28 @@ difference calculated between the start and end times:
      - From ``Thu, 28 Nov 2013 08:00:00 GMT`` to ``Thu, 28 Nov 2013 10:00:00 GMT``,
        with 1 hour resolution, will return 3 data points with metrics aggregated for each hour.
 
+Example::
+
+  POST '<base-url>/metrics/query?context=namespace.default.app.CountRandom&
+    metric=system.process.events.processed&start=now-1h&end=now&resolution=1m'
+
+This will return the value of the metric *system.process.events.processed* for the last
+hour at one-second intervals; potentially 61 entries may be returned, though only entries
```

Comment:
do we need this: "potentially 61 entries may be returned, though only entries
+where there is data will be returned."?